build.sh

```diff
@@ -53,6 +53,10 @@ install_rpms() {
     # Process our base dependencies + build dependencies and install
     (echo "${builddeps}" && "${srcdir}"/src/print-dependencies.sh) | xargs yum -y install
 
+    # Add fast-tracked packages here.  We don't want to wait on bodhi for rpm-ostree
+    # as we want to enable fast iteration there.
+    yum --enablerepo=updates-testing upgrade rpm-ostree
```

Comment:
not needed - I tagged the new rpm-ostree into `f35-coreos-continuous` earlier today.

Reply:
I think the idea is we want newer rpm-ostree releases earlier all the time, not just in this specific instance.

Reply:
ehh - I think that's reasonable but we'd probably want that to apply more broadly than just rpm-ostree right? Should we at least include ostree?

Reply:
I think we mostly get out of that by having rpm-ostree hard depend on newer ostree when we need it.  It's much less common to need a newer ostree, particularly given that most new logic there is in the Rust side which is vendored into rpm-ostree today.

So I think let's revisit adding more packages when we need to?

Reply:
Sounds good.